server/app/Models/Doc.php

```diff
@@ -411,6 +341,17 @@ public function content()
         return $this->hasMany('App\Models\DocContent');
     }
 
+    public function fullContentHtml()
+    {
+        return $this->content()
+            ->orderBy('page')
+            ->get()
+            ->reduce(function ($fullContent, $content) {
```

Comment:
We can't select the HTML from the database, so we have to get all the markdown content, convert it to HTML and then concatenate it. We could instead concat all the markdown content into one big string in the database then process that big string into HTML all at once. That could be more efficient. Hard to say without benchmarks. Either way, multiple large strings are going to be in memory.

Also I wasn't sure how much content is actually paginated currently/in the near future, everything I've seen so far has been a single page and I don't know if our story around multi-page documents is very good right now, so I'm defaulting to simple.

Note that this method is only called when generating the top level document RSS feed.